samples/MvcSample/Startup.cs

```diff
@@ -20,6 +20,13 @@ public void Configuration(IAppBuilder app)
                 //    @"Data Source=(LocalDB)\v13.0;Integrated Security=True;Initial Catalog=master",
                 //    "select 'a'"),
                 new PingHealthCheck(new PingHealthCheckOptions().AddHost("localhost", 1000)));
+
+            /* Sample with named checks
+            app.UseHealthChecks(
+                "/_health",
```

Comment:
Could this be uncommented to make both code paths accessible if this was changed to

```suggestion
                "/_health_named",
```

?

Reply:
Sure, I'll update with that.

I didn't update the logs to use the assigned names (though I think I'll fix that as well now that you mention it). The main usage for this is for usage with the Health Check UI package. Without this addition it looks like this.

![image](https://user-images.githubusercontent.com/875092/69967475-c5ca7680-1518-11ea-9fa5-d51b4a89ee61.png)

With it (with different names for checks so I can have multiple service or disk-checks as well) it looks like this.

![image](https://user-images.githubusercontent.com/875092/69967595-04f8c780-1519-11ea-8a93-e334d9a97f51.png)